edge-util/src/Microsoft.Azure.Devices.Edge.Util/edged/ApiVersion.cs

```diff
@@ -10,6 +10,7 @@ public sealed class ApiVersion
         public static readonly ApiVersion Version20191022 = new ApiVersion(3, "2019-10-22");
         public static readonly ApiVersion Version20191105 = new ApiVersion(4, "2019-11-05");
         public static readonly ApiVersion Version20200707 = new ApiVersion(5, "2020-07-07");
+        public static readonly ApiVersion Version20201010 = new ApiVersion(6, "2020-10-10");
```

Comment:
"2020-10-10" the correct version?

Reply:
It was long before I added this. Yes this is what I added for edgelet too.